wallet/src/repository/wallet/tests/mod.rs

```diff
@@ -1063,3 +1063,32 @@ fn test_create_vtt_with_locked_balance() {
         err,
     );
 }
+
+#[test]
+fn test_export_xprv_key() {
+    let (wallet, _db) = factories::wallet(None);
+
+    let password: types::Password = "password".to_string().into();
+    assert!(wallet
+        .export_master_key(password.clone())
+        .unwrap()
+        .starts_with("xprv"));
+    assert_eq!(
+        wallet
+            .export_master_key(password)
+            .unwrap()
+            .starts_with("xprvdouble"),
+        false
+    );
+}
+
+#[test]
+fn test_export_xprvdouble_key() {
+    let (wallet, _db) = factories::wallet_with_args(None, false);
```

Comment:
Add a small description to this tests:

```suggestion
    // Create a wallet that does not store the master key.
    // This is used to emulate a bug in previous versions of the wallet.
    // In that case, the exported master key format is not "xprv", it is "xprvdouble"
    let (wallet, _db) = factories::wallet_with_args(None, false);
```